src/app/dashboard/dashboard.component.scss

```diff
@@ -1,9 +1,34 @@
-.cards {
-    width: 1000px;
-    display: flex;
-    direction: row;
-    margin-top: 10px;
-    align-items: center;
-    justify-content: space-between;
-    background-color:#4B4B49;
-}
\ No newline at end of file
+.greetings {
+  height: 100%;
+  width: 100%;
+}
+
+.exoft-achievements {
+  height: 100%;
+  width: 100%;
+}
+
+.personal-achievements {
+  height: 100%;
+  width: 100%;
+}
+
+.thank-you {
+  height: 100%;
+  width: 100%;
+}
+
+.top-chart {
+  height: 100%;
+  width: 100%;
+}
+
+.badges {
+  height: 100%;
+  width: 100%;
+}
+
+.challenges {
+  height: 100%;
+  width: 100%;
+}
```

Comment:
these classes are duplicated. you can create single common class (.page-block for example) and use it
please change it :)

Reply:
fixed